Dockerfile

```diff
@@ -91,7 +91,7 @@ ENV PYTHONUNBUFFERED=TRUE
 ENV RUNNING_IN_DOCKER=TRUE
 
 # Make a static files directory
-RUN mkdir -p src/ui-build/static/admin
+RUN mkdir -p src/fidesctl/ui-build/static/admin
```

Comment:
On line 115 this directory is referenced as `/fides/src/ui-build/static/admin/`. It hink they should be the same? This one seems correct so maybe 115 needs to change?

Also it is just an empty directory here. Is that correct or does it need the files copied from `frontend` here also?

Reply:
whoa you're right. so I think we actually don't need this line anymore since the refactored code doesn't rely on the directory existing anymore. the frontend is actually built on L18, then copied over in L115. but you're totally right that L115 needs to be updated. will update and test

Reply:
OKAY SO it turns out in .dockerignore's you need to specify a full path, or a `**/path`. That actually masked this problem (since I had the `ui-build` locally). Both should be fixed now. Great finds @sanders41 !

relatedly, this is why I was getting such different numbers from you all in https://github.com/ethyca/fides/issues/887! the `node_modules` were not actually included in the .dockerignore, so mine were always ready to go. Now it's `**/node_modules` and so my build also takes longer now. mystery kinda solved 🕵️